lib/jupyter_on_rails/boot.rb

```diff
@@ -3,12 +3,12 @@
 boot_file = File.expand_path('config/boot.rb', root)
 require boot_file
 
-require_relative 'iruby_kernel_extention'
-JupyterOnRails::IRubyKernelExtention.root = root
-
 require 'iruby'
-module IRuby
-  class Kernel
-    prepend JupyterOnRails::IRubyKernelExtention
-  end
+IRuby::Kernel.after_initialize do
+  original = Dir.pwd
+  Dir.chdir root
+  app_file = File.expand_path('config/application.rb', root)
+  require app_file
+  Rails.application.require_environment!
+  Dir.chdir original
```

Comment:
Just driving by and noticed this PR. FYI, you can use the block form of `Dir.chdir` so you don't have to worry about cleaning up afterwards.

```suggestion
  Dir.chdir(root) do
      app_file = File.expand_path('config/application.rb', root)
	  require app_file
	  Rails.application.require_environment!
  end
```